container/options.go

```diff
@@ -216,6 +245,23 @@ func SplitPercent(p int) SplitOption {
 	})
 }
 
+// SplitFixed sets the size as a fixed value of the available space.
+// When using SplitVertical, the provided size is applied to the new left
+// container, the new right container gets the reminder of the size.
+// When using SplitHorizontal, the provided size is applied to the new top
+// container, the new bottom container gets the reminder of the size.
+// The provided value must be a positive number in the range 0 <= cells.
+// If not provided, defaults to DefaultSplitFixed.
```

Comment:
Can we also add a sentence here (and in the comment for SplitPercent) saying that only one can be used?

Something like:

// Only one of SplitFixed() and SplitPercent() can be specified.